src/vaadin-text-field-mixin.html

```diff
@@ -571,6 +610,12 @@
       this.shadowRoot.querySelector('[name="input"], [name="textarea"]')
         .addEventListener('slotchange', this._onSlotChange.bind(this));
 
+      this._slottedHint = this.querySelector(`[slot="hint"]`);
```

Comment:
Isn't this same logic as in `_hintSlotChange`? You could invoke `this._hintSlotChange()` here instead.

Reply:
Done